extension-client/src/api/index.ts

```diff
@@ -20,6 +20,11 @@ const schematicsV1 = require('@ibm-cloud/ibm-schematics/dist/schematics/v1');
 import * as auth from '../auth/auth';
 import * as type from '../type/index';
 import * as util from '../util';
+import * as shell from '../command/shell';
+import { posix } from 'path';
```

Comment:
`posix` import is not used anywhere in the code

Reply:
Those imports have now been removed